Jenkinsfile

```diff
@@ -39,7 +39,6 @@ pipeline {
           options { timeout(time: 120, unit: 'MINUTES') }
           parallel {
             stage('Conformance (LLVM)') { steps {                                         sh 'make test-conformance -j4 TEST_CONCRETE_BACKEND=llvm'      } }
-            stage('Proofs (Java)')      { steps { lock("kevm-java-${env.NODE_NAME}")    { sh 'make test-prove       -j2 TEST_SYMBOLIC_BACKEND=java'    } } }
```

Comment:
Do you know if the Jenkinsfile is currently being used? Could we remove it altogether?

Reply:
It's used for the master release job.